src/sonic-yang-models/yang-models/sonic-vxlan.yang

```diff
@@ -0,0 +1,212 @@
+module sonic-vxlan {
+    yang-version 1.1;
+    namespace "http://github.com/Azure/sonic-vxlan";
+    prefix svxlan;
+
+    import ietf-yang-types {
+        prefix yang;
+    }
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-extension {
+        prefix sonic-ext;
+    }
+    import sonic-vlan {
+        prefix svlan;
+    }
+	import sonic-types {
+		prefix stypes;
+	}
+
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description
+        "SONIC VXLAN";
+
+    revision 2021-04-12 {
+        description
+            "First revision.";
+    }
+
+    container sonic-vxlan {
+
+        container VXLAN_TUNNEL {
+
+            description "config db VXLAN_TUNNEL table";
+
+            list VXLAN_TUNNEL_LIST {
+
+                key "name";
+                max-elements 1;
+
+                leaf name {
+                    must "(starts-with(../name, 'vtep'))"
+                    {
+                        error-message "Error: Name should start with vtep";
+                        error-app-tag invalid-vtep-name;
+                    }
+                    type string {
+                        length 1..10;
+                    }
+                }
+
+                leaf src_ip {
+                    type inet:ipv4-address;
+                }
+            }
```

Comment:
I believe we also have "dst_ip" in vxlan tunnel. Please update
VXLAN_TUNNEL|{{tunnel_name}} 
    "src_ip": {{ip_address}} 
    "dst_ip": {{ip_address}} (OPTIONAL)

Reply:
I dont think, we support static vxlan tunnel yet, so dst_ip is not required.

Reply:
The dst_ip is not handled here. 
Even for non BGP P2P tunnel support the configuration will continue to have only the src_ip.